macos/xcode/FluentUI.xcodeproj/project.pbxproj

```diff
@@ -399,7 +399,7 @@
 			indentWidth = 4;
 			sourceTree = "<group>";
 			tabWidth = 4;
-			usesTabs = 0;
+			usesTabs = 1;
```

Comment:
no we shouldn't use tabs

Reply:
I have no opinion on what we "should" use.  However, all of the current files but one use tabs.  Setting spaces as the project-wide default causes Xcode to start insterting space indentation in files that currently use tabs, leading to mixed white space.  So we either recognize that tabs is the de facto standard for this project, or set an override in every file that uses tabs, or convert everything from tabs to spaces.

Reply:
Office devmain is tab recommended. for this open source project, and especially for swift files we use spaces.